README.md

```diff
@@ -118,15 +118,18 @@ It accepts a `Booster` struct (which is the output of `EXGBoost.train/2`).
 preds = EXGBoost.train(X, y) |> EXGBoost.predict(X)
 ```
 
-# Requirements
+## Requirements
+
+If you are contributing to the library and need to compile locally or choose to not use the precompiled libraries, you will need the following:
 
-If you choose to not use the precompiled libraries, you will need the following:
 * Make
 * CMake
 * If MacOS: `brew install libomp`
 
 When you run `mix compile`, the `xgboost` shared library will be compiled, so the first time you compile your project will take longer than subsequent compilations.
 
+You also need to set `CC_PRECOMPILER_PRECOMPILE_ONLY_LOCAL=true` before the first local compilation, otherwise you will get an error related to a missing checksum file.
```

Comment:
The local development workflow isn't that smooth yet. I managed to run everything ok, but I always get a warning regarding the checksum being missing.

Reply:
Thanks for adding this